docs/Сети связи/Web/Web-сайт и Web-страница.md

```diff
@@ -0,0 +1,98 @@
+# Web-сайт и Web-страница
+## Что такое Web-страница?
+
+Web-страница — это документ, предназначенный для распространения в Интернете.
+Она представляет собой обычный текстовый файл, имеющий расширение html.
+В этом файле содержится код Web-страницы — текст, являющийся ее содержимым,
+который отформатирован с помощью специальных тегов, заключенных в "<>".
+Примеры тегов:
+- H1, H2, … H6 — заголовки разного уровня
+- HR — горизонтальная линия
+- OL, содержащий в себе теги Li — нумерованный список
+- UL, содержащий в себе теги Li — маркированный список
+- B — __жирный__
```

Comment:
[ch-md-error-bold-style](https://chaberch.github.io/how-to-chaberch/markdown/#ch-md-error-bold-style)